src/data-workspace/section-form/section.module.css

```diff
@@ -5,6 +5,14 @@
     height: 100% !important;
 }
 
+@media print {
+    .labelWrapper {
```

Comment:
I added these because otherwise the titles of sections are very hard to read when printing, because the gradient background is not printed by default so the text is grey on white.

<img width="618" alt="image" src="https://user-images.githubusercontent.com/1014725/173793199-708f5137-f204-46af-b95b-3d15a80677f1.png">

<img width="372" alt="image" src="https://user-images.githubusercontent.com/1014725/173793400-b7a880e5-ab84-4147-8c02-f59af160b74d.png">